game_eggs/steamcmd_servers/dont_starve/egg-don-t-starve-together (1).json

```diff
@@ -1,13 +1,18 @@
 {
```

Comment:
why is there a (1) in the json name?

Reply:
Had to fix one missed config pull in the install script and didn't notice a duplicate file.